subsys/net/ip/tcp.c

```diff
@@ -1960,6 +1960,33 @@ static bool tcp_validate_seq(struct tcp *conn, struct tcphdr *hdr)
 		(net_tcp_seq_cmp(th_seq(hdr), conn->ack + conn->recv_win) < 0);
 }
 
+static int32_t tcp_compute_new_length(struct tcp *conn, struct tcphdr *hdr, size_t len)
+{
+	int32_t new_len = 0;
+
+	if (len > 0) {
+		/* Cases:
+		 * - Data already received earlier: len > 0 , new_len <= 0
```

Comment:
Could rewrite the comment: all len > 0 are obvious here since we are in a if statement that needs it.

Reply:
Fixed